.github/workflows/ci.yml

```diff
@@ -133,6 +137,25 @@ jobs:
           password: ${{ secrets.DOCKER_PASS }}
           push: true
 
+      # Only continue if we are releasing
+      # Login to GAR to publish production image
+      - name: get the GCP auth token
+        if: ${{ steps.should_build.outputs.result == 'true' }}
+        id: gcp-auth
+        uses: google-github-actions/auth@v2
+        with:
+          token_format: access_token
+          service_account: ${{ secrets.GAR_PUSHER_SERVICE_ACCOUNT_EMAIL }}
+          workload_identity_provider: ${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}
+
+      - name: login to GAR
+        if: ${{ steps.should_build.outputs.result == 'true' }}
+        uses: docker/login-action@v3
+        with:
+          registry: us-docker.pkg.dev
```

Comment:
Where is that documented that we should be using that registry ? (I'm unfamiliar with GAR, is that the default or one we chose to use explicitly ?)

Reply:
We chose explicitly. It's a question for @bqbn and I can add it in a follow up PR.

Reply:
`pkg.dev` to GCP is like `hub.docker.com` to Docker Inc., which means if you use GAR, you pretty much just use that name.

That said, there is some difference. GCP GAR has multiple domain names for multiple locations. The `us` in the name represents the US multi-region. You can read https://cloud.google.com/artifact-registry/docs/repositories/repo-locations for more details.